### ISSUE TITLE
feat: Identify when connections are being closed or crashing constantly 

### ISSUE THREAD

**Issue by viniarck** [kytos-ng/of_core#101]
Problem:

Network operators who are deploying Kytos-ng in production and using `of_core` need to be able to identify when OpenFlow connections aren't getting stable either because of packets/handshake or a generalized crashes. Our python runtime shouldn't not struggle handling connections as long as it's a reasonable value, if it is, then `of_core` should expose that this is happening (maybe through and endpoint) just so this can be used externally to spun up and switchover to a different `kytosd` instance, this can help for recoverable errors. 

Other than that, outside of code related implementation, network operators should also have alerts for how many errors or tracebacks have happened overtime, we can have this readily available on ES with Kibana, although alerts are premium ES feature, but the data is there, so a script could also poll or query that:

![20230215_150853](https://user-images.githubusercontent.com/1010796/219115915-cbf4c35c-4ac0-4e49-9db0-7793ecb7c6df.png)

![20230215_150842](https://user-images.githubusercontent.com/1010796/219115933-910f17ca-7150-4504-937f-d5f633decf6b.png)

cc'ing @italovalcy for his info

This issue still needs further discution, but overall that's the problem we need to solve.

**Comment by italovalcy** [comment]
I agree, @viniarck. This feature can be part of a watchdog Napp or something like this, which consolidates all validations (not only of_core) and translates into an operational status (which could indicate success, failure, or partial failure - includingg failure in non-critical components, so on)
